### PR TITLE
HTTPClient added with GET and POST 

### PR DESCRIPTION
**Pull request by CharithW**

### CODE REVIEW

File: HTTPClient.c [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* Declaring the variables.
+*/
+unsigned int nextTime = 0;    // Next time to contact the server
+int count = 0 ;  // Count of how many times loop went through
+HttpClient http;
+http_request_t request;
+http_response_t response;
+http_header_t headers[] = {
+
+      { "Content-type", "application/x-www-form-urlencoded" },
```
Comment (rehrumesh) [inline HTTPClient.c]: Required formatting.

Comment (CharithW) [reply HTTPClient.c]: Sure . Will work on it :)

**Comment by charithsoori** [comment]
@CharithW It seems readme has few conflicts :) . Get an update from master to your repo and do the commit .

**Comment by CharithW** [comment]
@charithsoori  Resolved the conflicts

**Pull request merged**